Algorithmia/insights.py

```diff
@@ -0,0 +1,13 @@
+import requests
+import json
+
+
+class Insights:
+    # Example of correct insights:
+    # {
+    #   {"aKey":"aValue"},
+    #   {"aKey2":"aValue2"}
+    # }
+    def __init__(self, insights):
+        # TODO should we get the URL from a config?
```

Comment:
I would like to see this use an environment variable with a fallback to localhost. First of all, when doing local dev, 9000 will often already be taken by play server (but this is expecting to hit a sidecar right?). Also, it's difficult to get users to update python client on a schedule, so having more flexibility from the start is valuable.

Reply:
I created this Jira to set the env variable. https://algorithmia.atlassian.net/browse/INSIGHTS-36. This code has been updated to default to the localhost:9000 if it is not set.